src/components/stepper/stepper.tsx

```diff
@@ -52,6 +54,14 @@ export class Stepper {
    */
   @Prop({ reflect: true }) numberingSystem?: NumberingSystem;
 
+  @Watch("numberingSystem")
+  numberingSystemChange(): void {
+    for (const item of this.itemMap.keys()) {
+      // trigger item rendering to pick up parent numbering system
+      forceUpdate(item);
```

Comment:
can we avoid forceUpdating the whole component and parse down the `numberingSystem` prop onto `stepper-item` and mark it as `internal`?

Reply:
Similar to [this](https://github.com/Esri/calcite-components/pull/6563/files#diff-7d3850db7c4d6df6d974db7f03ca7c67b4cfadc998eae076b508c7b78b266167R58) . Thoughts?

Reply:
Do you mean adding an internal `numberingSystem` prop to `stepper-item` and setting it to `stepper`'s `numberingSystem`? I think I tried doing that for `date-picker` when I first implemented `numberingSystem` but I don't remember the issue I faced. @jcfranco do you have any insights on this? For reference, I just yoinked this code his PR #6563

Reply:
Oh I see you updated his PR, I thought I was going crazy for a second there lol. Does the test I wrote in this PR work in the branch with your updates?

Reply:
haha yeah i updated it. yup the test included in here works in the PR https://github.com/Esri/calcite-components/pull/6563

Reply:
i added a test in there which kind of runs through all the supported numberingSystem. Looking back at it now, i think it is a overkill.

Reply:
Yeah also we don't want to use `numberStringFormatter.numberFormatOptions` in the test because we want to make sure it's working for our users and that's an internal API. I'd replace the test in #6563 with the one in this PR and then Franco can decide which one he wants to install. Thanks for digging into this!

Reply:
Sounds good. will update the test in #6563.